permutive/index.js

```diff
@@ -44,6 +44,9 @@ const getUser = () => {
  * Retrieve content from Ads v2
  */
 const getContent = (contentId) => {
+	if (contentId) {
+		return Promise.resolve();
```

Comment:
Are you resolving without a value? I am not sure I understand how this works

Reply:
Oh! Well spotted, thanks @carlesandres ! Going to make the change!

Reply:
Solved in https://github.com/Financial-Times/alphaville-ui/pull/14/commits/18d077cc448f189c4a70b0db5d5488d1300da686